packages/eslint-plugin/src/rules/action-rule.ts

```diff
@@ -0,0 +1,79 @@
+import { Rule } from "eslint";
+import { CallExpression, Identifier, Literal, VariableDeclarator, ArrowFunctionExpression, Node } from "estree";
+import { isIdentifier, isLiteral } from "../lib";
+
+type ActionCallExpression =
+    CallExpression
+    & { callee: Identifier, arguments: [] | [Literal] | [ArrowFunctionExpression] | [ArrowFunctionExpression, Literal] }
+type ActionVariableDeclarator = VariableDeclarator & { id: Identifier, init: ActionCallExpression }
+
+export const actionRule: Rule.RuleModule = {
+    meta: {
+        type: 'suggestion',
+        docs: {
+            description: "Add name for every action call"
+        },
+        fixable: 'code'
+    },
+    create: function (context: Rule.RuleContext): Rule.RuleListener {
+        return {
+            VariableDeclarator: d => {
+                if (!isActionVariableDeclarator(d)) return;
+
+                const initArgs = d.init.arguments;
+
+                if (initArgs.length === 0) {
+                    context.report({
+                        message: `action name is not defined`,
+                        node: d,
+                        fix: fixer => fixer.replaceText(d.init, `action("${d.id.name}")`)
+                    });
+                    return;
+                }
+
+                if (isLiteral(initArgs[0]) && initArgs[0].value !== d.id.name) {
+                    context.report({
+                        message: `action name is defined bad`,
```

Comment:
this could be like: `action "{{ varName }}" should be named as it's variable name, rename it to "{{ varName }}"`